m4_sessions/1706-inning/group_five.md

```diff
@@ -1,9 +1,50 @@
 # Group Five
 
-### Session Title:
+### Session Title: How and When to Learn a New Language?
 
 ### Session Outcomes: 
+- Create a repo of resources and tutorials for learning new languages and technologies that students can contribute to.
+- Provide outline of our process and experiences
+- Provide space for people to share their own challenges and solutions.
 
 ### Session Outline:
 
-### Names of group members:
+#### Whole Group discussion
+
+##### 10 minute introduction, 
+
+##### 15 min intro
+
+
+- Deliberate Practice
+  - Process of learning
+    - ID a specific subset
+    - Focus on performance in that subset
+    - Get feedback
+    - Update 
+
+   - Motivations
```

Comment:
Maybe add something specific about how learning more languages impacts your career?  Should you learn every language at a basic level or a few at a deeper level?